src/app/complex/complex-details/complex-properties/pdb-crossreferences/pdb-crossreferences.component.scss

```diff
@@ -0,0 +1,8 @@
+tbody {
+  height: 350px;
```

Comment:
A bit unsure of those constant px size, is it working well in all viewing contexts?

Reply:
I don't know where it is exactly coming from, but the PDB xrefs table already had a min size, so even if we only had 1 xref, the table will be the full height to match the pdb widget on the left side. From what I've seen, this is working fine.